addons/hr_expense/models/hr_expense.py

```diff
@@ -848,8 +848,12 @@ def _default_journal_id(self):
 
     @api.model
     def _default_bank_journal_id(self):
+        company_journal_id = self.env.company.company_expense_journal_id
+        if company_journal_id:
+            return company_journal_id
         default_company_id = self.default_get(['company_id'])['company_id']
-        return self.env['account.journal'].search([('type', 'in', ['cash', 'bank']), ('company_id', '=', default_company_id)], limit=1)
+        journal = self.env['account.journal'].search([('type', 'in', ['cash', 'bank']), ('company_id', '=', default_company_id)], limit=1)
```

Comment:
I guess, here it should be also `('type', 'in', ['cash', 'bank', 'general'])`; same goes got the field `bank_journal_id`.

Reply:
i need to decide on this, with accounting input indeed